Makefile

```diff
@@ -336,6 +336,12 @@ eve-%: pkg/%/Dockerfile build-tools $(RESCAN_DEPS)
 	@$(DOCKER_GO) "dep ensure -update $(GODEP_NAME)" $(dir $@)
 	@echo Done updating $@
 
+oldimages:
+	./tools/oldimages.sh
+
+imageclean:
```

Comment:
nit - wondering docker-image-clean would be a better target name. We are only deleting the docker images.. And not any others we download..

Reply:
Sure. I’ll get a new light PR for that.